torch/csrc/jit/passes/onnx/function_substitution.cpp

```diff
@@ -2,11 +2,84 @@
 
 #include <torch/csrc/jit/jit_log.h>
 #include <torch/csrc/jit/passes/onnx/helper.h>
+#include <torch/csrc/jit/passes/onnx/naming.h>
 
 namespace torch {
 namespace jit {
 
+namespace {
+
+const std::string top_module_variable_name = "";
```

Comment:
k for constant?

Reply:
It's between this (snake case convention) or k with camel case. I'm not too sure about k with snake case, something like `k_top_module_variable_name` is misleading. Going with k with camel case, and I think we will need to move away from camel case naming conventions for functions as they are too similar.

Reply:
Yeah I was suggesting kCamelCase name. Moving away from camel case for functions sounds like the right thing to do. We just need to do that consistently at least across the torch onnx namespaces.